fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/data/ElectionHandler.java

```diff
@@ -92,6 +94,32 @@ public static void handleElectionResult(HandlerContext context, ElectionResult e
     lao.updateElection(election.getId(), election);
   }
 
+  /**
+   * Process an OpenElection message.
+   * @param context the HandlerContext of the message
+   * @param openElection the message that was received
+   */
+  public static void handleOpenElection(HandlerContext context, OpenElection openElection)
+      throws DataHandlingException {
+    LAORepository laoRepository = context.getLaoRepository();
+    Channel channel = context.getChannel();
+
+    Log.d(TAG, "handleOpenElection: channel " + channel);
+    Lao lao = laoRepository.getLaoByChannel(channel);
+    Election election = laoRepository.getElectionByChannel(channel);
+
+    if (election.getState() != CREATED) {
```

Comment:
I think this should be set in line 61 during the setup, instead of: 
election.setEventState(OPENED);
or where is the state set to CREATED ?

Reply:
When the election Setup message is received, the election is set to CREATED. When it receives the OpenElection, it sets to OPENED. It made more sense to me.

Reply:
That's also what I understood from your code, but then why is it set to OPENED on line 61 in handleElectionSetup() ? It looks like a bug to me. And by the way, the naming consistency of the functions would be better respected with handleElectionOpen().

Reply:
It should be CREATED on the line 61, at least it is on local. Strange it didn't push. Thanks for clarifying.